File: Smart Start final/src/Entite/Postulation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entite;

/**
 *
 * @author Loua
 */
public class Postulation {
    int ID;
    int idFreelancer;
    int idAnnonce;
    String Contenu;
    public Postulation(){}
    public Postulation(int ID, int idFreelancer, int idAnnonce, String Contenu) {
        this.ID = ID;
        this.idFreelancer = idFreelancer;
        this.idAnnonce = idAnnonce;
        this.Contenu = Contenu;
    }

    public Postulation(int ID, int idFreelancer, int idAnnonce) {
        this.ID = ID;
        this.idFreelancer = idFreelancer;
        this.idAnnonce = idAnnonce;
    }
    

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public int getIdFreelancer() {
        return idFreelancer;
    }

    public void setIdFreelancer(int idFreelancer) {
        this.idFreelancer = idFreelancer;
    }

    public int getIdAnnonce() {
        return idAnnonce;
    }

    public void setIdAnnonce(int idAnnonce) {
        this.idAnnonce = idAnnonce;
    }

    public String getContenu() {
        return Contenu;
    }

    public void setContenu(String Contenu) {
        this.Contenu = Contenu;
    }

    @Override
    public String toString() {
        return "Postulation{" + "ID=" + ID + ", idFreelancer=" + idFreelancer + ", idAnnonce=" + idAnnonce + ", Contenu=" + Contenu + '}';
    }
    
    
    
}
    

File: MOBILE/MOBILE INDIV/smartstart_test/src/AnnonceBundle/Entity/Postulation.php
<?php

namespace AnnonceBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Postulation
 *
 * @ORM\Table(name="postulation", indexes={@ORM\Index(name="idFreelancer", columns={"idFreelancer"}), @ORM\Index(name="idAnnonce", columns={"idAnnonce"})})
 * @ORM\Entity(repositoryClass="AnnonceBundle\Repository\AnnonceRepository")
 */
class Postulation
{
    /**
     * @var integer
     *
     * @ORM\Column(name="ID", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="Contenu", type="text", length=65535, nullable=false)
     */
    private $contenu;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateAjout", type="datetime", nullable=false)
     */
    private $dateajout = 'CURRENT_TIMESTAMP';

    /**
     * @var \Annonce
     *
     * @ORM\ManyToOne(targetEntity="Annonce")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idAnnonce", referencedColumnName="ID")
     * })
     */
    private $idannonce;

    /**
     * @var \Freelancer
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idFreelancer", referencedColumnName="id")
     * })
     */
    private $idfreelancer;

    /**
     * Postulation constructor.
     * @param \DateTime $dateajout
     */

    public function __construct()
    {
        $this->dateajout = new \DateTime ();
    }

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getContenu()
    {
        return $this->contenu;
    }

    /**
     * @param string $contenu
     */
    public function setContenu($contenu)
    {
        $this->contenu = $contenu;
    }

    /**
     * @return \DateTime
     */
    public function getDateajout()
    {
        return $this->dateajout;
    }

    /**
     * @param \DateTime $dateajout
     */
    public function setDateajout($dateajout)
    {
        $this->dateajout = $dateajout;
    }

    /**
     * @return \Annonce
     */
    public function getIdannonce()
    {
        return $this->idannonce;
    }

    /**
     * @param \Annonce $idannonce
     */
    public function setIdannonce($idannonce)
    {
        $this->idannonce = $idannonce;
    }

    /**
     * @return \Freelancer
     */
    public function getIdfreelancer()
    {
        return $this->idfreelancer;
    }

    /**
     * @param \Freelancer $idfreelancer
     */
    public function setIdfreelancer($idfreelancer)
    {
        $this->idfreelancer = $idfreelancer;
    }


}